assets/js/scenes.js

```diff
@@ -25,12 +25,13 @@ const generateScenes = () => {
     ]);
 
     onClick("start-text", () => {
-      go("game");
+      go("game", {tl: 10, score: 0, eggs: 0});
```

Comment:
Praise: Nice! I wouldn't have thought of passing these through here. For the time left variable, I would just recommend naming it `timeLeft` to avoid any ambiguity!

Reply:
Good point about naming - I should know better!